examples/src/main/java/io/dapr/examples/state/README.md

```diff
@@ -80,9 +80,25 @@ public class StateClient {
         retrievedMessagesMono.block().forEach(System.out::println);
   
         System.out.println("Deleting states...");
-  
+
+        System.out.println("Verify delete key request is aborted if an etag different from stored is passed.");
         // delete state API
-        Mono<Void> mono = client.deleteState(STATE_STORE_NAME, FIRST_KEY_NAME);
+        try {
+          Mono<Void> mono = client.deleteState(STATE_STORE_NAME, FIRST_KEY_NAME, "100", null);
```

Comment:
You can do a block right in the end of this line.

Reply:
good catch, updated